nemo_curator/utils/distributed_utils.py

```diff
@@ -1011,8 +1038,16 @@ def _write_to_jsonl_or_parquet(
     output_path: str,
     output_type: Literal["jsonl", "parquet"] = "jsonl",
     partition_on: Optional[str] = None,
+    compression: Optional[str] = None,
 ):
     if output_type == "jsonl":
+        SUPPORTED_COMPRESSIONS = {"gzip", None}
```

Comment:
Can we make this a global variable `SUPPORTED_JSON_COMPRESSIONS` and import that and use it across.